types/dataframe-js/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for dataframe-js 1.0
+// Project: https://github.com/Gmousse/dataframe-js#readme
+// Definitions by: Alzemiro Iago de Moraes Thomaz <https://github.com/alzemiro-iago>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
```

Comment:
```suggestion
```
it's no longer enforced, min version is now 3.8 (shortly to be 3.9), so this line can be removed